<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NormalCard</class>
 <widget class="QWidget" name="NormalCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>119</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="baseWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>119</width>
     <height>119</height>
    </rect>
   </property>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>119</width>
      <height>119</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="using_label">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>92</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="raiseWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>119</width>
     <height>119</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>119</y>
     <width>119</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
